chapters.md

```diff
@@ -25,19 +25,13 @@ The `EditionFlagHidden Flag` behaves similarly to the `ChapterFlagHidden Flag`:
 
 #### EditionFlagDefault
 
-It is recommended that only one Edition's `EditionFlagDefault Flag` is set to `true`. If more then one `Default Edition` is present, the first `Default Edition` MUST be used.
-If an `EditionFlagDefault Flag` is set to `true`, this `Edition` MUST be used. When all `EditionFlagDefault Flags` are set to `false`, the first Edition MUST be used.
-
+It is RECOMMENDED that no more than one `Edition` have an `EditionFlagDefault Flag` set to `true`. The first `Edition` with the `EditionFlagDefault Flag` set to `true` is the `Default Edition`. When all `EditionFlagDefault Flags` are set to `false`, then the first `Edition` is the `Default Edition`.
 
 #### EditionFlagOrdered
 
-The `EditionFlagOrdered Flag` is a significant feature.
-
-If the `EditionFlagOrdered Flag` is set to `false`, "Matroska Simple-Chapters" are used and only the `ChapterTimeStart` of a chapter is used as chapter mark to jump to the predefined point in the timeline.
-Some chapter elements must be now ignored by the playback application. All these elements are now informational only.
+The `EditionFlagOrdered Flag` is a significant feature as it enables an `Edition` of `Ordered Chapters` which defines and arranges a virtual timeline rather than simply labeling points within the timeline. For example, with `Editions` of `Ordered Chapters` a single `Matroska file` can present multiple edits of a film without duplicating content. Alternatively if a videotape is digitized in full, one `Ordered Edition` could present the full content (including colorbars, countdown, slate, a feature presentation, and black frames), while another `Edition` of `Ordered Chapters` can use `Chapters` that only mark the intended presentation with the colorbars and other ancilary visual information excluded. If an `Edition` of `Ordered Chapters` is enabled then the Matroska Player MUST play those Chapters in their stored order from the timecode marked in the `ChapterTimeStart Element` to the timecode marked in to `ChapterTimeEnd Element`.
 
-If the `EditionFlagOrdered Flag` is set to `true`, the "Matroska Ordered-Chapters" feature is activated.
-A playback application must now also read the `ChapterTimeEnd Element`, and a new virtual timeline is used.
+If the `EditionFlagOrdered Flag` is set to `false`, `Simple Chapters` are used and only the `ChapterTimeStart` of a `Chapter` is used as Chapter mark to jump to the predefined point in the timeline. With `Simple Chapters`, a Matroska Plyaer MUST ignore certain `Chapter Elements`. All these elements are now informational only.
```

Comment:
typo: Player

Reply:
personally not sure that `used as Chapter mark` and `a Matroska Player` should have capital C and P…

Reply:
I agree to Reto.
Chapter mark -> chapter mark
Matroska Player -> Matroska player (What is wrong with "playback application"?)

Reply:
agree with lowercase chapter. 

Matroska Player is used elsewhere in this part of the spec. However at the top level, under Security Concerns,`Matroska Reader` is used and backticked. In the cues.md section, "Matroska reader" is used. We should pick and standardize this. "playback application" is also used in a few spots other than here.

Reply:
The EBML document defines "EBML Reader" in the Notations and Conventions section, along with other phrases used within the document. I'm inclined to consistently use "Matroska Reader" also. 

Definition:
"EBML Reader": An "EBML Reader" is a data parser that interprets the
   semantics of an "EBML Document" and creates a way for programs to use
   "EBML".

https://datatracker.ietf.org/doc/draft-ietf-cellar-ebml/?include_text=1

Reply:
I don't want to hold up this PR with this issue though -- should I ask the CELLAR list?

Reply:
Keep in mind that every reading application is not a player (e.g. mkvmerge), and that every issue is not a playback issue. Meaning that two different wordings "reader" and "player" actually make sense.

Here's an example:

All `Matroska readers` MUST handle default values properly.

All `Matroska players` must play `Edition` so-and-so under circumstances so-and-so.

Reply:
True, thanks for that point.

Reply:
With this new info I think `Matroska Player` for a playback application is fine, but `Matroska Player` sounds to specific so I would prefer playback application.
`Matroska Reader` for the reading applications sounds good.

Reply:
I prefer the generic "playback application" too!